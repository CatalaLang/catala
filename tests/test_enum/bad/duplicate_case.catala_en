## Article

```catala
declaration enumeration E:
  -- Case1 content integer
  -- Case2 content boolean
  -- Case3

declaration scope A:
  context e content E
  context out content boolean

scope A:
  definition out equals match e with pattern
    -- Case1 of i : i = 0
    -- Case2 of b : b
    -- Case3 : false
    -- Case3 : true
```
