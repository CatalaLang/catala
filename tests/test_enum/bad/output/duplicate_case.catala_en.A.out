[ERROR] The constructor Case3 has been matched twice:
[ERROR] 
[ERROR]   --> test_enum/bad/duplicate_case.catala_en
[ERROR]    | 
[ERROR] 18 |     -- Case3 : true
[ERROR]    |                ^^^^
[ERROR]    + Article
[ERROR] 
[ERROR]   --> test_enum/bad/duplicate_case.catala_en
[ERROR]    | 
[ERROR] 17 |     -- Case3 : false
[ERROR]    |                ^^^^^
[ERROR]    + Article
